lib/chef/provider/service/macosx.rb

```diff
@@ -1,3 +1,4 @@
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
```

Comment:
It seems weird to start checking these in - can this be put in some file local to your system?